lib/utils/api_exception_mapper.dart

```diff
@@ -1,22 +1,16 @@
-import 'package:smoge/app/strings.dart';
+import 'package:flutter/cupertino.dart';
 import 'package:smoge/data/api/api_exception.dart';
+import 'package:smoge/generated/l10n.dart';
 
 abstract class ApiExceptionMapper {
-  static String toErrorMessage(Object error) {
+  static String toErrorMessage(BuildContext context, Object error) {
```

Comment:
kontekst jest tu przekazywany dlatego, że bierzemy pod uwagę możliwość zmiany locale wewnątrz aplikacji?

Reply:
m.in. https://github.com/DroidsOnRoids/smoge-flutter/pull/10#discussion_r519461863